### ISSUE TITLE
"T" suffix in squota and hquota doesn't work

### ISSUE THREAD

**Issue by sathieu** [rozofs/rozofs#147]
I have this in exports.conf:
```
exports = (
# ...
   {
      eid = 2;
      bsize = "4K";
      root = "/srv/rozofs/exports/elasticsearch-snapshots";
      squota = "16T";
      hquota = "16T";
      vid=1;
    },
# ...
}
```

But "16T" is changed to 16, as seen in rozodiag :
```
fstat_thread eid 2
____[exportd-M]__[fs eid 2]____
file statistics for eid 2(0x14fd8c0) :/srv/rozofs/exports/elasticsearch-snapshots/fstat_0
 number of blocks (mem) : 0
 number of files  (mem) : 0
   nb files per number of block
 number of blocks (disk): 0
 number of files  (disk): 0
 hardware quota         : 16
 software quota         : 16
 quota exceeded         : NO

```

The doc has:
```
              for squota and hquota:
                      - quotas are express in nb. of blocks (no suffix), Kilo, Mega,
                        Giga or Tera Bytes according to suffix K, M, G or T. Only the
                         first letter of  suffix is used that is: 128G is equivalent to
                        128Giga which in turn can be 128GigaBytes etc...
                      - warning: any other suffix leads to quota express in blocks.
```

- "T" suffix is actually not supported
- "any other suffix leads to quota express in blocks." is not a good ideo IMO

**Comment by sathieu** [comment]
Note that "16384G" works:
```
fs eid 2
____[exportd-M]__[fs eid 2]____
file statistics for eid 2(0x2395690) :/srv/rozofs/exports/elasticsearch-snapshots/fstat_0
 number of blocks (mem) : 0
 number of files  (mem) : 0
   nb files per number of block
 number of blocks (disk): 0
 number of files  (disk): 0
 hardware quota         : 4294967296
 software quota         : 4294967296
 quota exceeded         : NO
```

**Comment by sathieu** [comment]
I've already done something similar in 46cf189d7d55e8c5a3d41e91354ff13c9a2638fe (v1).

**Comment by rozodopamine** [comment]
Fixed in release  2.7.3
